delivery.yaml

```diff
@@ -0,0 +1,37 @@
+build_steps:
+  - desc: Prepare
+    cmd: |
+      curl -o /tmp/ensure-docker -f https://delivery.cloud.zalando.com/utils/ensure-docker
+      chmod +x /tmp/ensure-docker
+      /tmp/ensure-docker
+
+  - desc: Build
+    cmd: |
+      function npm {
+        docker run -i --rm -v "$(pwd):/workspace" -w /workspace registry.opensource.zalan.do/stups/node:latest npm "$@"
```

Comment:
Could you just explain me what this line is doing? some docker magic for me :D

Reply:
Ok I see that this is used from the zappr part which as far as I understand to get the latest node + npm version. 
I should ask Rasha and others if we couldn't add this registry directly to docker and get rid of this line.